### DISCUSSION TITLE
where is glew.c file?

### DISCUSSION THREAD

**Discussion by Dahab000** [nigels-com/glew#327] [Q&A] [answered]
I am trying to compile glew with visual studio 2019 and I got this message ".\..\src\glew.c': No such file or directory"

**Comment by nigels-com** [comment, marked as answer]
It is highly recommended to build from a tgz or zip release snapshot. The code generation workflow is a complex brew of gnu make, perl and python, that works best on Linux or Mac. The code generation is known to work on Windows using MSYS2. For most end-users of GLEW the official releases are the best choice, with first class support.

https://github.com/nigels-com/glew#downloads

**Comment by orbitgw** [comment]
You can use the CMake to generate a VS2019 project and build the lib.
